src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java

```diff
@@ -809,7 +840,7 @@ private StudentExam generateIndividualStudentExam(Exam exam, User student) {
     @Transactional // TODO: NOT OK --> remove @Transactional
     public List<StudentExam> generateStudentExams(final Exam exam) {
         final var existingStudentExams = studentExamRepository.findByExamId(exam.getId());
-        // https://jira.spring.io/browse/DATAJPA-1367 deleteInBatch does not work, because it does not cascade the deletion of existing exam sessions, therefore use deleteAll
+        // deleteInBatch does not work, because it does not cascade the deletion of existing exam sessions, therefore use deleteAll
         studentExamRepository.deleteAll(existingStudentExams);
```

Comment:
This has nothing to do with our Jira instance, but instead links to a Spring issue.

Reply:
I think it's ok and we don't need the link to the JIRA Ticket